### DISCUSSION TITLE
Fastly CDN blocks access to GitHub

### DISCUSSION THREAD

**Discussion by woffic1al** [community/community#54068] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Select Feature Area

- [X] Issues
- [ ] Projects

### Body

Hi, I'm getting an error (500) when opening a GitHub website: Fastly error: unknown domain: github.com. Please check that this domain has been added to a service.

Details: cache-fra-eddf8230123-FRA

But when I want to open - https://resources.github.com/devops/fundamentals/devsecops/ then I catch bad requests (400)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @wezzyofficial ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. 
=
